tests/queries/0_stateless/02481_array_join_with_map.sql

```diff
@@ -0,0 +1,29 @@
+DROP TABLE IF EXISTS arrays_test;
+
+CREATE TABLE arrays_test
+(
+    s String,
+    arr1 Array(UInt8),
+    arr2 Array(UInt8), 
+    map1 Map(UInt8, String),
+ map2 Map(UInt8, String)
```

Comment:
Do we support `Nullable(Array(...))`/`Nullable(Map(...))`,`LowCardinality(Map(...))` and so on? It's good to add test cases as well.  And also maybe with some consts.

Reply:
ok. I'll add more tests later.

Reply:
@vdimir Why we are going to test `Nullable(Map(...))`, `LowCardinality(Map(...))`, it is not allowd in clickhouse.